src/client/src/ui/spotTile/components/PriceButton.tsx

```diff
@@ -102,18 +105,28 @@ const PriceButtonComp: React.FC<PriceButtonProps> = ({
   disabled = false,
 }) => {
   const bigFigure = getBigFigureDisplay(big, rawRate)
-  return (
-    <TradeButton direction={direction} onClick={() => handleClick()} disabled={disabled}>
-      <Price>
-        <BigWrapper>
-          <DirectionLabel>{direction.toUpperCase()}</DirectionLabel>
-          <Big>{renderBigFigureDisplay(bigFigure)}</Big>
-        </BigWrapper>
-        <Pip>{renderPips(pip)}</Pip>
-        <Tenth>{tenth}</Tenth>
-      </Price>
-    </TradeButton>
-  )
+  if (!displayDashAsDefaultPrice(big, pip, tenth, rawRate)) {
+    return (
+      <TradeButton direction={direction} onClick={() => handleClick()} disabled={disabled}>
```

Comment:
This needs to be kept DRY. This seems like a very tactical fix. I think you need to look at a more holistic solution.

Reply:
```  {rawRate ? (
        <Price>
          <BigWrapper>
            <DirectionLabel>{direction.toUpperCase()}</DirectionLabel>
            <Big>{renderBigFigureDisplay(bigFigure)}</Big>
          </BigWrapper>
          <Pip>{renderPips(pip)}</Pip>
          <Tenth>{tenth}</Tenth>
        </Price>
      ) : (
        <Price>
          <Pip>-</Pip>
        </Price>
      )}```